YOLayout/YOLayout_IMP.h

```diff
@@ -308,4 +314,14 @@ typedef CGSize (^YOLayoutBlock)(id<YOLayout> layout, CGSize size);
  */
 + (YOLayout *)layoutWithLayoutBlock:(YOLayoutBlock)layoutBlock;
 
+#pragma mark Common Layouts
+
++ (YOLayout *)vertical:(id)view;
+
++ (YOLayout *)fill:(id)view;
+
++ (YOLayoutBlock)verticalLayout:(id)view;
```

Comment:
Is it necessary to expose these blocks publicly?  Probably enough to not expose these in the header and just have the user use `vertical:` and `fill:` to get predefined `YOLayout`s